content/assets/stylesheets/_base.scss

```diff
@@ -13,3 +13,33 @@
 .reference-body h3+h4 {
   margin-top: 1em;
 }
+
+.scroll-top {
+  outline: none;
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+  position: fixed;
+  bottom: 50px;
+  right: 50px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.05);
+  overflow: hidden;
+  text-indent: 100%;
+  white-space: nowrap;
+  background: #218AC9 url(../../images/top-arrow.svg) no-repeat center 50%;
```

Comment:
Best to use fonts for icons, fontawesome tolarated but best is http://styleguide.bookingsync.com/brand/#iconography feel free to ask for more if missing github.com/BookingSync/bootstrap-bookingsync-sass